app/helpers/tasklist_helper.rb

```diff
@@ -29,7 +29,8 @@ def first_page_or_check_answers(subsection, case_log)
     path = if subsection.is_started?(case_log)
              "case_log_#{subsection.id}_check_answers_path"
            else
-             "case_log_#{subsection.pages.first.id}_path"
+             next_page = subsection.pages.first.routed_to?(case_log) ? subsection.pages.first.id : case_log.form.next_page(subsection.pages.first, case_log)
```

Comment:
This looks like it probably wants a spec?

Reply:
Hm, I thought it would already have one, lemme check

Reply:
It looks like you've added functionality for a new use case (the first page in the section isn't routed to), so should probably add a test for that specifically?

Reply:
You're right

Reply:
Added one now